### PR TITLE
Cythonize reify

### PR DESCRIPTION
**Pull request by asvetlov**
Another 5% boost on simple aiohttp web server

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,8 @@
 .vimrc
 aiohttp/_frozenlist.c
 aiohttp/_frozenlist.html
+aiohttp/_helpers.c
+aiohttp/_helpers.html
```
Comment (webknjaz) [inline .gitignore:27]: @asvetlov it looks like you forgot to update MANIFEST.in.
I wish we could just use a setuptools plugin, which would rely on git to do this automatically.

File: .travis.yml [hunk 2]
```diff
@@ -19,7 +19,8 @@ script:
 - make cov-ci-run
 
 after_success:
-- codecov
+ # - codecov
+ - bash <(curl -s https://codecov.io/bash)
```
Comment (webknjaz) [inline .travis.yml:23]: why do this? we install it in CI pinned to certain version:
https://github.com/aio-libs/aiohttp/blob/master/requirements/ci-wheel.txt#L25

File: .gitignore (hunk 1 above)
Comment (asvetlov) [reply .gitignore:27]: `*.c` files should be included but `*.html` should not.
Not very big problem though: on travis html files are never generated.

File: .travis.yml (hunk 2 above)
Comment (asvetlov) [reply .travis.yml:23]: It was my try to fix coverage reports.
Now upload on codecov is failed (with both codecov plugin and bash script).
Any ideas?

File: .gitignore (hunk 1 above)
Comment (asvetlov) [reply .gitignore:27]: Fixed by ffa4532e

**Pull request merged**